tests/helpers/kafka.ts

```diff
@@ -0,0 +1,82 @@
+import { EventEmitter } from 'events'
+import { Kafka, Consumer, logLevel, EachMessagePayload, Producer } from 'kafkajs'
+import { KAFKA_EVENTS_INGESTION_HANDOFF } from '../../src/ingestion/topics'
+import { parseRawEventMessage } from '../../src/ingestion/utils'
+import { EventMessage, RawEventMessage } from '../../src/types'
+import { UUIDT } from '../../src/utils'
+
+export class KafkaObserver extends EventEmitter {
+    public kafka: Kafka
+    public producer: Producer
+    public consumer: Consumer
+
+    private isStarted: boolean
+
+    constructor() {
+        super()
+        this.kafka = new Kafka({
+            clientId: `plugin-server-test-${new UUIDT()}`,
+            brokers: process.env.KAFKA_HOSTS!.split(','),
```

Comment:
Can we get this from the `server` somehow? Using `process.env` feels dirty, even if it's just in a test.

Reply:
Not really, because this is supposed to be entirely independent from `server` (the way ClickHouse is).